SPECS/pango/pango.spec

```diff
@@ -1,13 +1,13 @@
 Summary:        library for laying out and rendering of text.
 Name:           pango
-Version:        1.44.7
-Release:        3%{?dist}
+Version:        1.45.5
```

Comment:
I am ok with the bump, but there could be downstream consequences.  Not sure how many dependent packages there are, but please make sure that anything dependent still builds.

Reply:
Sure, that makes sense.